__tests__/integration/test-utils-integration.js

```diff
@@ -0,0 +1,349 @@
+    addresses: [
+      'WY1URKUnqCTyiixW1Dw29vmeG99hNN4EW6', // Genesis funds
+      'WRTFYzhTHkfYwub8EWVtAcUgbdUpsYMBpb', // Miner rewards
+      'WhpJeUtBLrDHbKDoMC9ffMxwHqvsrNzTFV', // First without transactions
+    ]
+  },
+  second: {
+    walletId: 'secondwallet',
+    words: 'scare more mobile text erupt flush paper snack despair goddess route solar keep search result author bounce pulp shine next butter unknown frozen trap',
+    addresses: [
+      'WTjhJXzQJETVx7BVXdyZmvk396DRRsubdw',
+      'Wdf7xQtKDNefhd6KTS68Vna1u4wUAyHjLQ',
+      'WaQf5igKpbdNyxTBzc3Nv8a8n4DRkcbpmX',
+    ]
+  },
+}
+
+export const HATHOR_TOKEN_ID = "00"
+
+export function getRandomInt(max, min = 0) {
+  return Math.floor(Math.random() * max) + min;
+}
+
+export class TestUtils {
+  /**
+   * Simple way to wait asynchronously before continuing the funcion. Does not block the JS thread.
+   * @param {number} ms Amount of milisseconds to delay
+   * @returns {Promise<unknown>}
+   */
+  static async delay(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms))
+  }
+
+  /**
+   * Returns the Supertest `request` object for this application
+   * @returns {Test}
+   */
+  static get request() {
+    return request;
+  }
+
+  /**
+   * Generates seed words for a new wallet
+   * @returns {string|*}
+   */
+  static generateWords() {
+    return wallet.generateWalletWords()
+  }
+
+  /**
+   * Generates the header for requesting a protected route on the wallet headless
+   *
+   * @see TestUtils.stopWallet
+   * @param {string} walletId
+   * @returns {{"x-wallet-id"}}
+   */
+  static generateHeader(walletId) {
+    return { "x-wallet-id": walletId }
+  }
+
+  /**
+   * Adds a message to the Transaction Log for a test
+   * @param message
+   */
+  static logTx(message) {
+    loggers.test.insertLineToLog(message)
+      .catch(err => console.error(err.stack))
+  }
+
+  /**
+   * Starts a wallet. Prefer instantiating a WalletHelper instead.
+   * @param {WalletData} walletObj
+   * @returns {Promise<{start:unknown,status:unknown}>}
+   */
+  static async startWallet(walletObj) {
+    let start, status
+
+    // Request the Wallet start
+    const response = await request
+      .post("/start")
+      .send({ seed: walletObj.words, "wallet-id": walletObj.walletId });
+
+    // Handle errors
+    if (response.status !== 200) {
+      throw new Error(`Unable to start the wallet: ${walletObj.walletId}`);
+    }
+    if (!response.body.success) {
+      console.error(`Failure starting the wallet: ${response.body.message}`)
+      throw new Error(response.body.message);
+    }
+    start = response.body
+
+    // Wait until the wallet is actually started
+    while (true) {
+      const res = await request
+        .get("/wallet/status")
+        .set(TestUtils.generateHeader(walletObj.walletId));
+      if (res.body && res.body.success !== false) {
```

Comment:
Improved this validation with an explicit comparison with a `"Ready"` string on `statusMessage`, instead of a non-intuitive `success` being not false.